ludwig/encoders/text_encoders.py

```diff
@@ -2169,6 +2170,8 @@ def __init__(
             self.reduce_sequence = SequenceReducer(reduce_mode=reduce_output)
         self.max_sequence_length = max_sequence_length
 
+        self.forward_kwargs = set(inspect.signature(self.transformer.module.forward).parameters.keys())
```

Comment:
nit: Could you add a comment here explaining why we're inspecting? Might be helpful down the road since this was motivated by a bloom-specific fix.

Reply:
Added comments.